src/main/java/org/asamk/signal/commands/UpdateGroupCommand.java

```diff
@@ -168,7 +177,13 @@ public void handleCommand(final Namespace ns, final Signal signal) throws Comman
         try {
             var newGroupId = signal.updateGroup(groupId, groupName, groupMembers, groupAvatar);
             if (groupId.length != newGroupId.length) {
-                writer.println("Created new group: \"{}\"", Base64.getEncoder().encodeToString(newGroupId));
+                String encodedGroup = Base64.getEncoder().encodeToString(newGroupId);
+                if (ns.get("output") == OutputType.JSON) {
+                    final var jsonWriter = new JsonWriter(System.out);
+                    jsonWriter.write(Map.of("group", encodedGroup, "members", groupMembers, "name", groupName));
```

Comment:
```suggestion
                    jsonWriter.write(Map.of("groupId", encodedGroupId));
```
I think outputting the groupId would be enough here, `groupMembers` and `groupName` are already provided by the user.

Reply:
if #668 is okay, I added these so that you don't need to keep state to associate group IDs with members and names.